ecosystem/typescript/sdk/src/aptos_types/transaction.ts

```diff
@@ -420,6 +422,12 @@ export abstract class TransactionArgument {
         return TransactionArgumentU8Vector.load(deserializer);
       case 5:
         return TransactionArgumentBool.load(deserializer);
+      case 6:
```

Comment:
There’s a transaction argument vec<u8> above- do we not allow vectors of the other native types?

Reply:
Double checked this. We do not allow vectors of the other native types.